test/duplicated-properties.js

```diff
@@ -0,0 +1,23 @@
+const test = require('ava');
+const testFactory = require('./_test-factory');
+const plugin = require('../src')({removeDuplicatedProperties: true});
```

Comment:
thanks for adding tests for the new deduplication functionality. :+1: 
Could some tests ensuring that properties are preserved when `removeDuplicatedProperties: false` be added as well?